module/zfs/zvol.c

```diff
@@ -1008,6 +1008,16 @@ zvol_request(struct request_queue *q, struct bio *bio)
 				zvol_write(zvr);
 		}
 	} else {
+		/*
```

Comment:
LGTM. I would `ASSERT3S(rw, ==, READ)` here but that's not really direcly related to this patch so feel free to leave as is